### ISSUE TITLE
multilayer SW solver requires dgesv

### ISSUE THREAD

**Issue by rjleveque** [clawpack/riemann#146]
Running nose tests on a new AWS EC2 instance revealed this issue in `rpn2_layered_shallow_water.f90`

lapack is required or at least the subroutine `dgesv` and any dependencies:

https://github.com/clawpack/riemann/blob/3d89c52f0c6ae6ed4e5ac3102b457b1931a8b592/src/rpn2_layered_shallow_water.f90#L103

**Comment by mandli** [comment]
Are you thinking we should fall-back to directly compiled source for the routine?  We definitely do not want to usually do this.

**Comment by rjleveque** [comment]
Yes, best to use lapack if available.  But we could provide something like we do for this example:
https://github.com/clawpack/apps/blob/master/fvmbook/chap17/advdiff/tridiag.f

**Comment by mandli** [comment]
So we would need to come up with some sort of check to see if we can link against a native LAPACK.  Obviously a configure could do this so we probably need to come up with something similar.

**Comment by rjleveque** [comment]
Or should we just cleanly skip this unit test if lapack isn't available, as happens with the netCDF tests?

**Comment by mandli** [comment]
Might be easiest.
